modules/study/src/main/StudyApi.scala

```diff
@@ -207,6 +215,10 @@ final class StudyApi(
 
   def kick(studyId: Study.Id, userId: User.ID) = sequenceStudy(studyId) { study =>
     study.isMember(userId) ?? {
```

Comment:
ah if it's a Funit, it makes sense. I thought it was an unboxed side effect (Unit).